runtime/syntax/man.vim

```diff
@@ -18,6 +18,18 @@ highlight default link manOptionDesc     Constant
 highlight default link manReference      PreProc
 highlight default link manSubHeading     Function
 
+function! s:init_highlight_groups()
+  highlight default manUnderline cterm=underline gui=underline
+  highlight default manBold      cterm=bold      gui=bold
+endfunction
+
+augroup man_init_highlight_groups
+  autocmd!
+  autocmd ColorScheme * call s:init_highlight_groups()
```

Comment:
why is this needed? `highlight default` is a one-time thing.

Reply:
I needed it in order to keep the highlighting across colorscheme changes. AFAICT `highlight default link`s are preserved after a `highlight clear`, but groups (other than the builtin ones) with defined default attributes get cleared. Am I missing something?

Reply:
Does this happen on startup, or only if user changes colorscheme at runtime? The latter is not important. And why wouldn't this also be needed for the other `hi default`'s, e.g. on line 19:

    highlight default link manSubHeading     Function

More generally, syntax needs to be re-initialized if a colorscheme or whatever nukes it. Every plugin cannot be expected to deal with that ad-hoc.